addons/default/visiosoft/advs-module/src/Support/Command/CheckModuleInstalled.php

```diff
@@ -0,0 +1,23 @@
+<?php namespace Visiosoft\AdvsModule\Support\Command;
+
+use Anomaly\Streams\Platform\Addon\Module\Contract\ModuleRepositoryInterface;
+
+class CheckModuleInstalled
```

Comment:
We can check this addon because there are extensions, themes, plugins outside the module.

Reply:
For a comprehensive project we need